packages/core/package.json

```diff
@@ -1,14 +1,20 @@
 {
   "name": "pcln-design-system",
-  "version": "2.6.2",
+  "version": "2.7.0",
   "description": "Priceline Design System",
-  "main": "dist/index.cjs.js",
-  "module": "dist/index.esm.js",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "sideEffects": false,
```

Comment:
I think apps that have a server-side implementation of react-loadable may run into an issue with this.

Reply:
Perhaps you could make a prerelease version so I can confirm if there are breaking changes.